### PR TITLE
Update end-user-mam-apps-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a section for required approved client app

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1357

File: memdocs/intune/fundamentals/end-user-mam-apps-ios.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ If the device is **not enrolled in Intune**, the user is asked to restart the ap
 
 For devices that are **enrolled for management in Intune**, the user sees a message that their app is now managed.
 
+
+### Require approved client app
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/end-user-mam-apps-ios.md:46]:
```suggestion
### Approved client app requirement
```

File: memdocs/intune/fundamentals/end-user-mam-apps-ios.md [hunk 2]
```diff
@@ -42,6 +42,14 @@ If the device is **not enrolled in Intune**, the user is asked to restart the ap
...
+
+Organizations can require that an access attempt to the selected cloud apps needs to be made from an approved client app. These approved client apps support [Intune app protection policies](/intune/app-protection-policy) independent of any mobile-device management (MDM) solution.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/end-user-mam-apps-ios.md:48]:
```suggestion
Organizations might require that an access attempt to the selected cloud apps needs to be made from an approved client app. These approved client apps support [Intune app protection policies](/intune/app-protection-policy) independent of any mobile device management (MDM) solution.
```

File: memdocs/intune/fundamentals/end-user-mam-apps-ios.md [hunk 3]
```diff
@@ -42,6 +42,14 @@ If the device is **not enrolled in Intune**, the user is asked to restart the ap
...
+
+In order to apply this grant control, Conditional Access requires that the device is registered in Azure Active Directory, which requires the use of a broker app. The broker app can be the Microsoft Authenticator for iOS. If a broker app isn’t installed on the device when the user attempts to authenticate, the user gets redirected to the appropriate app store to install the required broker app.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/end-user-mam-apps-ios.md:50]:
```suggestion
In order to apply this grant control, Conditional Access requires that the device is registered in Azure Active Directory, which requires the use of a broker app. The broker app could be the Microsoft Authenticator for iOS. If a broker app isn’t installed on the device when the user attempts to authenticate, the user gets redirected to the appropriate app store to install the required broker app.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and merge this PR. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**